gaia-flex/config/genesis.json

```diff
@@ -2604,5 +2619,5 @@
       ]
     }
   },
-  "genesis_time": "2020-08-17T12:00:00.000000000Z"
```

Comment:
Any ideas why this is changing @ethanfrey?

Reply:
I think I made the other one by hand. The round down is more correct